spring-cloud-gcp-autoconfigure/src/main/java/com/google/cloud/spring/autoconfigure/trace/StackdriverTraceAutoConfiguration.java

```diff
@@ -150,24 +158,22 @@ public ManagedChannel stackdriverSenderChannel() {
 				.build();
 	}
 
-	@Bean(REPORTER_BEAN_NAME)
-	@ConditionalOnMissingBean(name = REPORTER_BEAN_NAME)
-	public Reporter<Span> stackdriverReporter(ReporterMetrics reporterMetrics,
```

Comment:
Why not keep `stackdriverReporter` as a bean, and create an additional bean for `stackdriverSpanHandler` that depends on `stackdriverReporter`?

Reply:
You're right - reverting some of these changes and adding that bean back.